python-main.js

```diff
@@ -1124,47 +1124,94 @@ function web_editor(config) {
         $("#command-flash").attr("id", "command-download");
         $("#command-download > .roundlabel").text(config["translate"]["static-strings"]["buttons"]["command-download"]["label"]);
         $("#command-download").attr("title", config["translate"]["static-strings"]["buttons"]["command-download"]["title"]);
+
+        var p = Promise.resolve();
+
+        if (usePartialFlashing) {
+            if (window.dapwrapper) {
+                p = p.then(() => window.dapwrapper.daplink.stopSerialRead())
```

Comment:
IE10/11 do not support arrow functions and will throw an exception when trying to part `python-main.js`, even if the functions are not called.

Reply:
@microbit-sam could you look into all the ES6 features in python-main.js and change them as ES5 ones? We need to make sure the file parsing doesn't throw an exception in IE10/11.